projects/aducm_blinky_example/src.mk

```diff
@@ -0,0 +1,28 @@
+################################################################################
+#									       #
+#     Shared variables:							       #
+#	- PROJECT							       #
+#	- DRIVERS							       #
+#	- INCLUDE							       #
+#	- PLATFORM_DRIVERS						       #
+#	- NO-OS								       #
+#									       #
+################################################################################
+
+#In aducm projects the name of the source dir should be different from src
+#Direcory where app srcs are stored
+APP_SRCS = app_src
+
+SRC_FOLDERS +=	$(DRIVERS)/accel/adxl345
+#		$(NO-OS)/network 	 \
+		$(DRIVERS)/sd-card	 \
+		$(NO-OS)/libraries/fatfs
+
+#Include makefiles from each source directory if they exist
+DIRECORIES_WITH_MAKEFILES = $(wildcard $(foreach dir,$(SRC_FOLDERS),$(dir)\src.mk))
+include $(DIRECORIES_WITH_MAKEFILES)
+
+#For the folders with no makefiles include all *.c and *.h
+REMAINING_DIRECORIES = $(filter-out, $(DIRECORIES_WITH_MAKEFILES), $(SRC_FOLDERS))
+SRCS += $(foreach dir, $(REMAINING_DIRECORIES), $(wildcard $(dir)/*.c))
```

Comment:
Importing wildcards is a bad ideea

Reply:
The idea behind this is: 
If it is a folder with sources than should not be imported it can be used a src.mk in that folder with somethin like:
if (PLATFORM = xilinx)
   SRCS += ...
else
  SRCS += ....
This way you don't have to know in each project which files are needed for a driver.
Also, If we will agree with this aproach, I think this should be moved inside the makefile and let the user define SRC_FOLDERS only.

Reply:
Here https://github.com/analogdevicesinc/no-OS/tree/master/projects/adrv9009/profiles are the same sources for different profiles

Reply:
In that case src.mk can look like:
SRC_FOLDERS +=	$(PROJECT)/src\
                                $(PROJECT)/profiles/desired_profiles